app/src/main/java/org/schabi/newpipe/App.java

```diff
@@ -212,7 +212,7 @@ protected void initACRA() {
 
         try {
             final CoreConfiguration acraConfig = new CoreConfigurationBuilder(this)
-                    .setBuildConfigClass(BuildConfig.class)
+                    .withBuildConfigClass(BuildConfig.class)
                     .build();
```

Comment:
from the docs:
http://www.acra.ch/docs/Setup

instead of calling build() you can also directly provide the CoreConfigurationBuilder to ACRA.init

I would say we should take this opportunity to init it compliant to the docs

Reply:
```
            final CoreConfigurationBuilder builder = new CoreConfigurationBuilder(this)
                    .withBuildConfigClass(BuildConfig.class);
            ACRA.init(this, builder);
```

Reply:
Actually we can't do that because then `ACRAConfigurationException` isn't thrown. Should we get rid of the try catch block?

Reply:
well sounds good to me if we don't need the try catch anymore

Reply:
Okay.